config/comp_libs.in

```diff
@@ -92,6 +91,12 @@ config EXPAT_TARGET
 config NCURSES_NEEDED
     bool
     select COMP_LIBS_NCURSES
+    help
+      Add ncurses to the build if you want support for tui in gdb
+      or color support in the msgcat program of gettext
+
+      Be aware that enabling ncurses may make gdb compiled for
+      windows not usable.q
```

Comment:
Stray 'q'

Reply:
Thanks. Didn't see that. I updated the pull request